com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipelineAsset.cs

```diff
@@ -88,25 +88,12 @@ internal VolumeProfile defaultLookDevProfile
             set => m_DefaultLookDevProfile = value;
         }
 
+        [SerializeField]
```

Comment:
Please be extra carefull here. This is data only available in a UNITY_EDITOR context.
Can this cause crash while loading it in a player?
Can this cause HDRenderPipelineEditorResources  to leak into player ?

Reply:
Good point. I will ask QA to check the player.